thunar-gpg-encrypt.sh

```diff
@@ -110,11 +108,9 @@ chooseRecipient () {
 chooseSecret () {
 
 	seckeys="$(gpg --list-secret-keys \
-	  | grep -A 1 sec \
-	  | awk '{print $2,$3,$4}' \
-	  | egrep -v "^[[:space:]]*$" \
-	  | awk 'NR%2{printf $1" ";next;}1' \
-	  | awk '{print substr($0,7,8)" "substr($0,1,5)" \""$2,$3,$4"\""}')"
+	  | grep -A 1 "^pub" \
```

Comment:
`gpg --list-secret-keys` only shows the following three sections:
```shell
sec  xxxx/xxxx
uid                   name <mail>
ssv  yyyy/yyyy
```

Maybe you wanted to do `grep -A 1 "^sec"` ?